src/moapplication.cpp

```diff
@@ -81,12 +142,304 @@ MOApplication::MOApplication(int& argc, char** argv)
 
   m_DefaultStyle = style()->objectName();
   setStyle(new ProxyStyle(style()));
+  addDllsToPath();
+}
+
+int MOApplication::run(SingleInstance& singleInstance)

```

Comment:
Here as well, instead of single instance it could maybe be "processInstance"? There modded instance related stuff all over in these functions.